rudder-relay/SPECS/rudder-relay.spec

```diff
@@ -70,62 +70,25 @@ Requires: %(../format-dependencies rpm %{old_epoch}:%{real_version} rudder-agent
 
 ## RHEL
 %if 0%{?rhel}
-Requires: mod_ssl shadow-utils
+Requires: mod_ssl shadow-utils python3-policycoreutils
 BuildRequires: selinux-policy-devel
 %endif
 
-%if 0%{?rhel} && 0%{?rhel} > 7
-Requires: python3-policycoreutils
-%endif
-%if 0%{?rhel} && 0%{?rhel} <= 7
-Requires: policycoreutils-python
-%endif
-
 ## SLES
 %if 0%{?suse_version}
 Requires: pwdutils
 %endif
 
-# Doc for suse versioning https://en.opensuse.org/openSUSE:Packaging_for_Leap
-%if 0%{?suse_version} && 0%{?suse_version} < 1500
-BuildRequires: python
-Requires: python, python-pyOpenSSL
-%endif
-%if 0%{?suse_version} && 0%{?suse_version} >= 1500
-BuildRequires: python3
-Requires: python3
-%endif
-
-
 ## Python
-%if 0%{?rhel} == 7 || ( 0%{?suse_version} && 0%{?suse_version} < 1500 )
-BuildRequires: python, python-setuptools, python-lxml, python-requests
-Requires: python, python-setuptools, python-lxml, python-requests
-%else
 BuildRequires: python3, python3-pip, python3-lxml, python3-requests
 Requires: python3, python3-lxml, python3-requests, python3-setuptools
-%endif
 
 %description
 Rudder is an open source configuration management and audit solution.
 
 This package is essentially a meta-package to install all components required to
 run a Rudder relay server on a machine.
 
-#=================================================
-# Source preparation
-#=================================================
-%prep
-%setup -c
-
-# We don't know the exact version
-cd rudder-sources-*/rudder/relay/sources/
```

Comment:
why removing this?

Reply:
Indeed, but I think the setup -c is still required to correctly uncompress the source

Reply:
i thought it was the default

Reply:
the -c is not